_layouts/default.html

```diff
@@ -97,5 +97,32 @@
         </div>
         <link rel="stylesheet" type="text/css" href="plugins/cookie-consent/style.css" />
         <script src="plugins/cookie-consent/script.js"></script>
+        <script>
+          document.onreadystatechange = function () {
```

Comment:
Instead of "setting" a new function I think we need to "add" a callback?

Reply:
added an event listener on 'DOMContentLoaded' instead of setting document.onreadystatechange